File: AtCoder/abc204/B.kt
private fun readLn() = readLine()!!
private fun readLns() = readLn().split(" ")
private fun readInt() = readLn().toInt()
private fun readInts() = readLns().map { it.toInt() }

fun main() {
    val n = readInt()
    val arr = readInts()
    var ans = 0
    for (ele in arr) {
        ans += Math.max(0, ele - 10)
    }
    print(ans)
}

File: AtCoder/abc204/B.swift
extension String {
	var intValue: Int {
		return Int(self) ?? 0
	}
}

private func readLn() -> String { return readLine() ?? "" }
private func readLns() -> [String] {
	return readLn().split(separator: " ").compactMap({ String($0) })
}
private func readInt() -> Int {
	return readLn().intValue
}
private func readInts() -> [Int] {
	return readLns().map({ $0.intValue })
}

let n = readInt()
let arr = readInts()
var ans = 0
for ele in arr {
	ans += max(0, ele - 10)
}
print(ans)
